### ISSUE TITLE
Change "tout menu" title to something more descriptive

### ISSUE THREAD

**Issue by penguininja** [ryanaslett/bitchmedia#2]
I'll be using the menu and/or block titles as alternatives to the icons on mobile and tablet for accessibility purposes, so "tout menu" should be changed to a more descriptive name.

**Comment by ryanaslett** [comment]
This is changed, feel free to close this if it works.  It *might* have renamed/changed the tout menu css class. I cant quite figure out what regions everything is supposed to be in, context wise.
